code/script.js

```diff
@@ -32,16 +35,123 @@ const showMessage = (message, sender) => {
+    setTimeout(() => showMessage(`That's rude! Please give me your name.`, 'bot'), 1000)
+  }
+  else{ //If user puts in name... continue...
+    setTimeout(() => showMessage(`Hey ${name}!`, 'bot'), 1000) //bot says Hey to 'name'
+    setTimeout(() => drinkMenu(), 2000) // continues to drinkMenu
+  }
+}
+
+const drinkMenu = () => {
+  showMessage(`Whats your poison?`, 'bot')
+  //Create buttons for drinkMenu
+  inputWrapper.innerHTML = `
+  <button id='whiskeyButton'>Whiskey</button>
+  <button id='rumButton'>Rum</button>
+  <button id='vodkaButton'>Vodka</button>
+  `
+  //Whiskey button pressed
+  document.getElementById('whiskeyButton').addEventListener('click', () =>  {
+    showMessage('Whiskey', 'user')
+    setTimeout(() => showMessage(`Ah, Whiskey... good choice! How many glasses?`, 'bot'), 1000)
+    setTimeout(() => drinkNumberFunc(), 1000)
+    })
+
+  //Rum button pressed
+  document.getElementById('rumButton').addEventListener('click', () =>  {
+    showMessage('Rum', 'user')
+    setTimeout(() => showMessage(`Aye! Rum for my matey. How many glasses?`, 'bot'), 1000)
+    setTimeout(() => drinkNumberFunc(), 1000)
+    })
+
+  //Vodka button pressed
+  document.getElementById('vodkaButton').addEventListener('click', () =>  {
+    showMessage('Vodka', 'user')
+    setTimeout(() => showMessage(`Da! Vodka for my comrade. How many glasses?`, 'bot'), 1000)
+    setTimeout(() => drinkNumberFunc(), 1000)
+    })
+}
+
+//Drink number function
+const drinkNumberFunc = () => {
+  //Clears previous buttons and insert form field again
+  inputWrapper.innerHTML = `
+  <form id='name-form'>
+  <input id='drink-input' type='text'/>
+  <button class='send-btn' id='send-btn' type='submit'>
+          Send
+  </button>
+  </form>
+  `
+  //Stores value in drinkNumber, listens to if button gets pressed
+  document.getElementById('send-btn').addEventListener('click', (event) => {
+    event.preventDefault()
+    let drinkNumber = document.getElementById('drink-input').value
+    showMessage(drinkNumber, 'user')
+    document.getElementById('drink-input').value = ''
+    const audio = new Audio('./assets/Clink-sound.mp3') //For drink clink audio
+
+    //Depending on nr of drinks, diffrent bot answers
+    if ( drinkNumber === '' || drinkNumber === '0' ) {
+      setTimeout(() => showMessage("Can't order 0 drinks. Please say again.", 'bot'), 1000)
+      drinkNumberFunc()
+    }
+    else if ( drinkNumber > '0' && drinkNumber < '2'){
+      setTimeout(() => showMessage(`Alright, ${drinkNumber} glass coming up!`, 'bot'), 1000)
+      setTimeout(() => audio.play(), 2000) //Plays drink clink
+      //Image of glass in bot bubble
+      setTimeout(() =>
+      chat.innerHTML += `
+      <section class="bot-msg">
+        <img src="assets/bartender.jpeg" alt="Bot" />
+        <div class="bubble bot-bubble">
+        <img src="./assets/whiskey-glass.jpeg" id="glass" alt="glass" />
+        </div>
+      </section>
+    ` , 1000)
```

Comment:
this code is being repeat in row 130-139. Can we put this code in a function? And just invoke the function inside this else if-statement and else-statement? 🦊